src/core/mac/mac.cpp

```diff
@@ -481,23 +478,13 @@ void Mac::HandleBeginTransmit(void)
     }
 }
 
-extern "C" void otPlatRadioSignalTransmitDone(void)
+extern "C" void otPlatRadioTransmitDone(bool aRxPending, ThreadError error)
 {
-    sTransmitDoneTask.Post();
+    sMac->TransmitDoneTask(aRxPending, error);
 }
 
-void Mac::TransmitDoneTask(void *aContext)
+void Mac::TransmitDoneTask(bool aRxPending, ThreadError error)
```

Comment:
If you are going to prefix function/method args with "a" then you should do it consistently as in error == aError.